inc/sainexthopgroup.h

```diff
@@ -184,6 +184,41 @@ typedef enum _sai_next_hop_group_attr_t
      */
     SAI_NEXT_HOP_GROUP_ATTR_SELECTION_MAP,
 
+    /**
+     * @brief Adaptive routing and switching object for this group.
+     *
+     * @type sai_object_id_t
+     * @flags CREATE_AND_SET
+     * @objects SAI_OBJECT_TYPE_ARS
+     * @allownull true
+     * @default SAI_NULL_OBJECT_ID
+     */
+    SAI_NEXT_HOP_GROUP_ATTR_ARS_OBJECT_ID,
+
+    /**
```

Comment:
TCP-based traffic (sensitive to OOO) can be handled with flowlet ARS only. With a single ARS object per ECMP (non-sensitive to TCP or UDP) it looks like only flowlet ASR can be used. Correct ?